src/Compilers/Server/VBCSCompilerTests/VBCSCompilerTests.csproj

```diff
@@ -25,7 +25,6 @@
       <Aliases>vbc</Aliases>
     </ProjectReference>
     <ProjectReference Include="..\..\..\Test\PdbUtilities\PdbUtilities.csproj" />
-    <ProjectReference Include="..\..\..\Test\Utilities\Portable\TestUtilities.csproj" />
```

Comment:
The other reference is here: https://github.com/dotnet/roslyn/pull/22847/files#diff-10ddcb0bfab78fda8365fbe7a7efb09aL20. It's above the github-shown preview.